src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int LOTTERY_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < LOTTERY_BALLS; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/199219212-67b6368e-3b49-48d8-beca-540e5d4b1cec.png)
Do not create new Lottery() inside loop.